.spelling

```diff
@@ -479,6 +479,54 @@ embeddable
 p99
 vCPU
 AES-NI
+3.99ms
+72.96ms
+18x
+8x
+appswitch
+appswitch-0
+AppSwitch
+bitpipe
+DaemonSet
+datapath
+decap
+decapsulated
```

Comment:
unencapsulated

Reply:
I guess the distinction is decapsulated is verb whereas unencapsulated is an adjective.  Also encap / decap are accepted networking terms.